app/stac_api/views.py

```diff
@@ -152,13 +153,24 @@ def get_queryset(self):
         date_time = self.request.query_params.get('datetime', None)
 
         if bbox:
-            raise NotImplementedError('bbox query parameter not yet implemented')
+            queryset = self.filter_by_bbox(queryset, bbox)
 
         if date_time:
             queryset = self.filter_by_datetime(queryset, date_time)
 
         return queryset
 
+    def filter_by_bbox(self, queryset, bbox):
+        try:
+            logger.debug(bbox)
+            query_bbox_polygon = Polygon.from_bbox(bbox.split(','))
+        except:
```

Comment:
Never ever catch all exceptions ! Only catch known exceptions e.g. 'ValueError', see https://github.com/geoadmin/doc-guidelines/blob/master/PYTHON.md#7-error-handling---rules-of-thumb

Reply:
Thanks - I will aprove